sdks/java/io/elasticsearch-tests/elasticsearch-tests-2/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOIT.java

```diff
@@ -50,6 +50,9 @@
  *      "--elasticsearchServer=127.0.0.1",
  *      "--elasticsearchHttpPort=9200"]'
  * </pre>
+ *
+ * <p>It is likely that you will need to configure <code>thread_pool.bulk.queue_size: 250</code> (or
+ * higher) in Elasticsearch for this test to run.
  */
```

Comment:
this is a comment for infra guys make this explicit: in the backend Elasticsearch server targeted.